cluster/charts/crossplane/README.md

```diff
@@ -314,7 +314,8 @@ and their default values.
 | `resourcesPackageManager.requests.memory` | Memory resource requests for PackageManager                | `256Mi`
 | `forceImagePullPolicy`           | Force the named ImagePullPolicy on Package install and containers | `""`
 | `insecureAllowAllApigroups`      | Enable core Kubernetes API group permissions for Packages. When enabled, Packages may declare dependency on core Kubernetes API types.) | `false` |
-| `insecurePassFullDeployment`     | Enable packages to pass their full deployment, including security context. When omitted, Packages deployments will have security context removed and all containers will have `allowPrivilegeEscalation` set to false. | `false` |
+| `insecurePassFullDeployment`     | Enable packages to pass their full deployment, including security context. When omitted, Package controller deployments will have security context removed and all containers will have `privileged` and `allowPrivilegeEscalation` set to `false`, and `runAsNonRoot` set to `true`. | `false` |
+| `insecureInstallJob`     | Enable package install jobs to run as root. When omitted, Package install jobs will have security context removed and all will have `privileged` and `allowPrivilegeEscalation` set to `false`, and `runAsNonRoot` set to `true`. | `false` |
```

Comment:
is it missing `containers` completely now? `all will` -> `all containers will`, like it says in cmd/crossplane/package/manage/manage.go`

Reply:
hmm did find and replace here so not sure how it worked for the `cmd` but not `docs`. Oh well, will update again :man_facepalming: